Xamarin.Forms.Controls/CoreGalleryPages/FloatingActionButtonCoreGalleryPage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Xamarin.Forms.CustomAttributes;
+
+namespace Xamarin.Forms.Controls
+{
+	internal class FloatActionButtonCoreGalleryPage : CoreGalleryPage<FloatingActionButton>
+	{
+		protected override bool SupportsTapGestureRecognizer
+		{
+			get { return false; }
+		}
+
+		protected override bool SupportsFocus
+		{
+			get { return false; }
+		}
+
+		protected override void InitializeElement(FloatingActionButton element)
+		{
+			element.HorizontalOptions = LayoutOptions.Center;
```

Comment:
None of the specific FloatingActionButton properties are used here. Can we add ImageSource, Size and Command here?
```
Test. FloatingActionButton.ImageSource
Test. FloatingActionButton.Size
Test. FloatingActionButton.Command
```

Reply:
Added these properties!